src/themes/custom/app/request-copy/email-request-copy/email-request-copy.component.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<!--
+Copyright 2023 Indiana University.
```

Comment:
@mwoodiupui : Just a quick note... I was glancing at the code of this PR (haven't tested yet) and noticed this appears to contain some test code you were using.  :)   I think this HTML file is meant to be empty.

Reply:
Thanks for catching that.  Fixed.